src/js/emoji-picker.js

```diff
@@ -0,0 +1,237 @@
+/*!
+ * EmailOctopus EmojiPicker v1.0.0 (https://emailoctopus.com/)
+ * License MIT
```

Comment:
@jonathanbull - we should decide what license we're going to use (if we're going to open source it) - MIT is the most common one.

Reply:
MIT is good for me